src/WPF/WPF.Viewer/Samples/Analysis/QueryFeatureCountAndExtent/QueryFeatureCountAndExtent.xaml.cs

```diff
@@ -68,39 +90,32 @@ private async Task Initialize()
             }
         }
 
-        private async void BtnZoomToFeaturesClick(object sender, RoutedEventArgs e)
+        private async void ZoomToFeatures(object sender, RoutedEventArgs e)
         {
             try
             {
                 // Create the query parameters.
-                QueryParameters queryStates = new QueryParameters { WhereClause = $"upper(State) LIKE '%{StateTextbox.Text}%'" };
+                QueryParameters queryStates = new QueryParameters { WhereClause = $"State LIKE '%{_states[StatesComboBox.SelectedItem.ToString()]}%'" };
```

Comment:
I can see what you're doing here but maybe for the sake of clarity we could declare a string above like:

```
string selectedState = _states[StatesComboBox.SelectedItem.ToString()];
```

then

```
QueryParameters queryStates = new QueryParameters { WhereClause = $"State LIKE '%{selectedState}%'" };
```

I think this makes it more obvious to the user exactly what our query is doing.